homeassistant/components/wemo.py

```diff
@@ -91,6 +92,7 @@ def discovery_dispatch(service, discovery_info):
         if device is None:
             device = pywemo.discovery.device_from_description(url, None)
 
-        discovery_info = (device.name, device.model_name, url, device.mac)
+        discovery_info = (device.name, device.model_name, url, device.mac,
```

Comment:
I might be missing something, but this tuple is just generated by this module and passed internally to something like wemo.switch so it can configure itself, right? Is there some overhead in us passing a lot of info?

Regardless, I think that all of these pieces of info are indeed used by the dispatch method above, or switch.wemo/light.wemo in one form or another.